src/styles/cardStyles.scss

```diff
@@ -0,0 +1,108 @@
+.text-Main {
+  transition-duration: $duration;
+  color: $Main-accent;
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 18px;
+  margin: 40px 0 0;
+}
+
+.text-Secondary {
+  color: $Secondary;
+  font-size: 10px;
+  line-height: 14px;
+  margin: 4px 0 0;
+}
+
+.price {
+  display: flex;
+  justify-content: space-between;
+  margin-top: 24px;
+  line-height: 18px;
+
+  &__text {
+    color: $Secondary;
+    font-size: 12px;
+    font-weight: 400;
+  }
+
+  &__value {
+    color: $Main-accent;
+    text-align: right;
+    font-size: 16px;
+    font-weight: 700;
+  }
+}
+
+.rating {
+  display: flex;
+  margin-top: 16px;
+  width: 100%;
+  height: 16px;
+  justify-content: space-between;
+
+  &__text {
+    color: $Main-accent;
+    text-align: right;
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+    align-self: flex-end;
+  }
+}
+
+.button {
+  width: 166px;
+  height: 40px;
```

Comment:
it's not a good practice to hardcode height for elements with content inside. use padding instead
```suggestion
```